README.md

```diff
@@ -1,2 +1,17 @@
 # mesh-orange
-Image for a cjdns + IPFS mesh node running on the Orange Pi Zero + TOP-GS07 RT5572 WiFi adapter
+
+The following instructions will help set up a mesh network node running on the Orange Pi Zero & TOP-GS07 RT5572 WiFi adapter.
+
+# Set Up
+
+Make sure you have the following items:
+
+  * Orange Pi Zero
+  * TOP-GS07 RT5572 WiFi adapter
+  * An SD card that works with the Pi Zero
+
+## Download Armbian
+
+[Armbian](https://www.armbian.com/) is a debian-based linux distro for ARM development boards, check out their [Getting Started guide](https://docs.armbian.com/User-Guide_Getting-Started/) for more details.
+
+1. Download and flash the [Armbian Nightly image](https://www.armbian.com/orange-pi-zero/)
```

Comment:
What I did here was to install the Armbian Server Debian Jessie image from that link and then followed the instructions on the Kernel "dev - development (4.x)" tab to upgrade to the 4.10 kernel.  In order to get the more recent wireless tools, I also upgraded from jessie to stretch (I think I did this before the kernel upgrade)